Makefile.am

```diff
@@ -34,12 +36,13 @@ examplejpegdir = $(exampledir)/JPEG
 examplepngdir = $(exampledir)/PNG
 
 if JPEG
-examplejpeg_PROGRAMS = examples/JPEG/remove_red_from_rgb examples/JPEG/rgb_to_grayscale examples/JPEG/invert
+examplejpeg_PROGRAMS = examples/JPEG/remove_red_from_rgb examples/JPEG/rgb_to_grayscale examples/JPEG/invert \
+					   examples/JPEG/padImage examples/JPEG/edgeDetection
```

Comment:
JPEG examples do not contain edgeDetection file. It seems like you forgot to push that file.

Reply:
pushed